src/components/backend-ai-session-list.ts

```diff
@@ -1593,12 +1593,45 @@ export default class BackendAiSessionList extends BackendAIPage {
         ` : html``}
         <vaadin-grid-column width="40px" flex-grow="0" header="#" .renderer="${this._indexRenderer}"></vaadin-grid-column>
         ${this.is_admin ? html`
-          <vaadin-grid-sort-column resizable width="130px" header="${this._connectionMode === 'API' ? 'API Key' : 'User ID'}" flex-grow="0" path="access_key" .renderer="${this._boundUserInfoRenderer}">
-          </vaadin-grid-sort-column>
+          <vaadin-grid-column resizable width="270px" flex-grow="0" path="access_key" .renderer="${this._boundUserInfoRenderer}">
```

Comment:
Could we shorten the width of this column?
IMO, It seems too large. The average length of the text that values contain barely reaches the current length.
<img width="1552" alt="Screen Shot 2021-08-06 at 12 08 12 AM" src="https://user-images.githubusercontent.com/46954439/128374001-a721e6c6-0e7a-4486-92e8-96d60f278d42.png">

Please check the similar column header on the Data & Storage page.

Reply:
Or, we could set the alignment of the column name and input field from horizontal to vertical.

Reply:
<img width="551" alt="image" src="https://user-images.githubusercontent.com/26023759/128447380-27f3ece0-4993-4464-979b-34420139a6bd.png">
I've made column name and text field vertically because width of text-field increases automatically by length of contents on columns and i couldn't know how to adjust it.

Reply:
@romanceformoon please check the API documentation of `vaadin-grid-filter-column`. Here's the [link](https://vaadin.com/components/vaadin-grid/html-api/elements/Vaadin.GridFilterColumnElement).

Reply:
My prior comment was about `vaadin-text-field` with using `vaadin-grid-column`, 
I could not find to set width of `vaadin-text-field`. 😢

Reply:
As far as I know is that `vaadin-grid` component creates the textfield element internally when it renders.
And when it renders, the width of textfield is probably set by the width attribute of ` vaadin-grid-filter-column`.
So you should check whether setting width attribute (probably under 50px for significant changes) works or not.

Reply:
Also, It would be appreciated that you could tell the approaches that you tried solving these kinds of issues, next time. 😉